open-api/rest-catalog-open-api.yaml

```diff
@@ -3142,6 +3211,10 @@ components:
           type: object
           additionalProperties:
             type: string
+        storage-credentials:
+          type: array
```

Comment:
Need a definition which set of credentials a client shall pick, in case multiple credentials match.
Either we make it easy for clients and specify that a client must use the credentials with first matching prefix (the server can sort the credentials by prefix-length (descending), or we specify that a client must pick the credentials with the longest matching prefix.
I have a slight preference to the first option to reduce complexity and effort from "all the client implementations".

Reply:
this is already being mentioned on how the client should choose the credential (aka by the longest matching prefix)